pkg/util/utils.go

```diff
@@ -37,6 +37,12 @@ import (
 	"k8s.io/minikube/pkg/version"
 )
 
+type RetriableError struct {
+	Err error
```

Comment:
I think this would be a bit cleaner if you embed the error interface here, then you wouldn't need to redefine the Error() method below.